src/nvim/normal_cmds.lua

```diff
@@ -0,0 +1,1157 @@
+local bit = require('bit')
+
+local NONE = 0
+local NCH = 0x01
+local NCH_NOP = bit.bor(0x02, NCH)
+local NCH_ALW = bit.bor(0x04, NCH)
+local LANG = 0x08
+local SS = 0x10
+local SSS = 0x20
+local STS = 0x40
+local RL = 0x80
+local KEEPREG = 0x100
+local NCW = 0x200
+
+local SEARCH_REV = 0x01
+local BL_WHITE = 1
+local BL_FIX = 4
+
+local FORWARD = 1
+local BACKWARD = -1
+
+local TRUE = 1
+
+local MSCR_DOWN = 0
+local MSCR_UP = 1
+local MSCR_LEFT = -1
+local MSCR_RIGHT = -2
+
+local module = {}
+
+module.nv_cmds = {
+  {
+    cmd_char = 'NUL',
+    cmd_func = 'nv_error',
+    cmd_flags = NONE,
+    cmd_arg = NONE,
+  },
+  {
+    cmd_char = 'Ctrl_A',
+    cmd_func = 'nv_addsub',
+    cmd_flags = NONE,
+    cmd_arg = NONE,
+  },
+  {
+    cmd_char = 'Ctrl_B',
+    cmd_func = 'nv_page',
+    cmd_flags = STS,
+    cmd_arg = BACKWARD,
```

Comment:
Using `ex_cmds.lua` as a reference, and thinking about how this data will be used, I don't see a reason why we would need the `cmd_` prefix on the fields.

These should also be documented at the top of this file. A `@class` may be helpful: https://github.com/neovim/neovim/blob/f6a3fdd6848d67dc54cebb6c297f8ebdc109c3a3/src/nvim/eval.lua#L5

```suggestion
    char = 'Ctrl_B',
    func = 'nv_page',
    flags = STS,
    arg = BACKWARD,
```